packages/opentelemetry-exporter-zipkin/test/common/transform.test.ts

```diff
@@ -58,14 +58,14 @@ const spanContext: api.SpanContext = {
 describe('transform', () => {
   describe('toZipkinSpan', () => {
     it('should convert an OpenTelemetry span to a Zipkin span', () => {
-      const span = new Span(
-        tracer,
-        api.ROOT_CONTEXT,
+      const span = tracer.startSpan(
         'my-span',
-        spanContext,
-        api.SpanKind.SERVER,
-        parentId
-      );
+        { kind: api.SpanKind.SERVER },
+        api.ROOT_CONTEXT
+      ) as unknown as Span;
+      (span as any)['_spanContext'] = spanContext;
```

Comment:
note to reviewer: in some cases (like this one) it was necessary to set the `_spanContext` and the `parentId` because the assertions where on these internal props.

Reply:
I wonder if it would be even easier to just have `span` be of type `ReadableSpan` in this test. Since it's just an interface, we can directly create the object instead of going through the SDK. :thinking:

Reply:
This was the easier approach and also tests are calling `span.end()` so my first thought is that logic was needed. I'll give it a try :)

Reply:
@pichlermarc the PR changed completely 😅 but I think I got what you meant